.pre-commit-config.yaml

```diff
@@ -1,65 +1,65 @@
-repos:
```

Comment:
I'm not sure why this is changing or what has changed. Please remove from the commit.

Reply:
It was line endings.

I've removed it.